src/prometheus_alert_rules/reload_failure.rule

```diff
@@ -0,0 +1,8 @@
+alert: TraefikConfigReloadFailure
+expr: rate(traefik_config_reloads_failure_total[3m]) > bool 0
+for: 1m
+labels:
+  severity: critical
+annotations:
+  summary: Traefik config reload failure (instance {{ $labels.instance }})
+  description: "Traefik config reload failure rate is above 0"
```

Comment:
I think "failure rate is above 0" could be made more grokable for a busy on-caller. How about:

```suggestion
  description: "Traefik keeps failing to reload its config file"
```

(At some point we should write a styleguide for alerts.)

Reply:
> (At some point we should write a styleguide for alerts.)

You are reading my mind @sed-i !

I think we will also at some point need to curate our alerts and dashboards to have a unified UX.